src/Command/Dispatcher/DispatchFilesCommand.php

```diff
@@ -291,7 +294,6 @@ private function deleteNotNeededFilesAndDirs(Project $project, Branch $branch, s
                 ->toString();
 
             if ($this->filesystem->exists($file)) {
-                $this->io->writeln(sprintf('Delete <info>/%s</info> file!', $fileToRemove));
```

Comment:
This output is annoying because everytime it says what is going to be removed, even if its already removed. IMO not needed. If something is about to be removed it will show on the diff